nbqa/save_source.py

```diff
@@ -18,7 +18,7 @@
 if TYPE_CHECKING:
     from pathlib import Path
 
-CODE_SEPARATOR = "# %%"
+CODE_SEPARATOR = "# %%\n"
```

Comment:
It might be better to make this more 'nbqa unique', e.g. `# %%NBQA-CELL-SEP\n`.
I can imagine a case where `# %%\n` would end up in a real word notebook, e.g. the user doesn't remember the name of a cell magic on the spot and left the start of the cell magic as a comment, just as a reminder to use implement it later.

Reply:
that's a good point actually, thank you so much!